### ISSUE TITLE
Baseline details

### ISSUE THREAD

**Issue by ynw2021** [raphychek/mbappe-nuplan#1]
Hi,

Great works! How did you obtain the performances of the baseline urbandriver and gameformer in table 3 in your paper? Did your train on the train150k set from PDM that have 4000 scenarios per type? And what are the hyperparameters？

bests，

**Comment by raphychek** [comment]
Hi,
Thank for your interest in our research!
For GameFormer we used the trained weights the author used for the leaderboard. For UrbanDriver Multi-Agent, we trained it until convergence on 8% of the dataset (maximum we could do with our computational ressources) - and it actually gives very similar results than UrbanDriver trained on train150k (c.f. Val14 benchmark from PDM paper). 
For PDM, we also used the official weights from the author. 
We justify our choice first with the observation that PDM does not need more than the train150k set at training to reach its near-optimal metrics. The goal of the comparison is, to the extent of what we could actually train, to assess what given methods can achieve at near-optimal, and most importantly how this would be improved leveraging MBAPPE.
